Orange/tests/test_classification.py

```diff
@@ -8,11 +8,15 @@
 
 import traceback
 import numpy as np
+import Orange
```

Comment:
Seems to be via `Orange.evaluation.AUC` below. I don't know how that worked before, or how Python's imports work, obviously. :confused: